### PR TITLE
Adding additional recommendations

### PR DESCRIPTION
**Pull request by mortonjt**
Added a couple of comments on performing paired-end alignment and quality filtering that could help future users of this tool.

### CODE REVIEW

File: doc/wol.md [hunk 1]
```diff
@@ -56,8 +56,17 @@ bowtie2 -x databases/bowtie2/WoLr1 -p 16 -f input.fa -S output.sam -k 16 --np 1
 
 This step will generate a [SAM](https://en.wikipedia.org/wiki/SAM_(file_format)) format alignment file.
 
-**Note**: You can also run Bowtie2 manually using your choice of parameters, or using other aligners and corresponding databases. Woltka is designed for flexibility.
+**Note**: You can also run Bowtie2 manually using your choice of parameters, or using other aligners and corresponding databases. Woltka is designed for flexibility.  For instance, you can run Bowtie2 to handle paired ends reads as follows
```
Comment (qiyunzhu) [inline doc/wol.md:59]: Should be "paired-end reads".

**Review by qiyunzhu — CHANGES_REQUESTED** [review]
@mortonjt Thanks for contributing to the documentation! Looks good. Just a small typo.

File: doc/wol.md (hunk 1 above)
Comment (mortonjt) [inline doc/wol.md:59]:
```suggestion
**Note**: You can also run Bowtie2 manually using your choice of parameters, or using other aligners and corresponding databases. Woltka is designed for flexibility.  For instance, you can run Bowtie2 to handle paired-ends reads as follows
```

**Review by mortonjt — COMMENTED** [review]
Addressing typo

**Review by qiyunzhu — APPROVED** [review]
I think it is good to go now.

**Pull request merged**